app/assets/stylesheets/overrides.scss

```diff
@@ -98,9 +98,17 @@ height: 49px;
   background-color: $subnav;
   border-bottom-left-radius: 30%;
   border-bottom-right-radius: 30%;
-  font-size: 10pt;
-  height: 45px;
-  padding-top: 10px;
+  padding: 10px 20px;
```

Comment:
Properties should be ordered background-color, border-bottom-left-radius, border-bottom-right-radius, height, padding